grpc-server/grpc_server.go

```diff
@@ -60,14 +60,19 @@ func SetupGRPC(ctx context.Context, logger log.Logger, config *ConfigGRPCServer,
 		dbReady: true,
 		logger:  logger,
 	}
-	if cert := config.TLSCert; cert != "" {
```

Comment:
Could we add a field to `ConfigGRPCServer` that specified no TLS?

The original `if` statement might work as-is... But it does leave some ambiguity about what should happen if the field is an empty string.

Reply:
I'll be reworking this soon and should do a better job of dealing with tls certs. Lets ignore this for now please.